ldap-authenticator/src/main/java/org/xwiki/contrib/ldap/internal/LDAPGroupsCache.java

```diff
@@ -101,7 +101,18 @@
      */
     public LRUCacheConfiguration createCacheConfiguration(XWikiLDAPConfig config)
     {
-        LRUCacheConfiguration cacheConfiguration = new LRUCacheConfiguration(CACHE_NAME_GROUPS);
+        return createCacheConfiguration(config, null);
+    }
+
+    /**
+     * @param config the current LDAP configuration
+     * @param cacheKeySuffix an extra suffix to make the cache key unique, if not null
+     * @return the cache configuration
+     */
+    protected LRUCacheConfiguration createCacheConfiguration(XWikiLDAPConfig config, String cacheKeySuffix)
```

Comment:
Why not private ?

Reply:
I am not sure if someone uses a derived class of this one.
If this is not a supported use case, I will make it private

Reply:
It's not really a supported use case no, the reason the other one is public is just because it's used by the long deprecated `XWikiLDAPUtils#getGroupCacheConfiguration()`.

Reply:
ok, made it private